correlation_context/HTTP_HEADER_FORMAT.md

```diff
@@ -1,43 +1,69 @@
 # Correlation Context HTTP Header Format
 
-A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `traceparent`. The values should be passed along to any child requests. Note that uniqueness of the key within the `Correlation-Context` is not guaranteed. Context received from upstream service may be altered before passing it along.
+The correlation context header is used to propagate user-supplied key-value pairs through a trace.
```

Comment:
There still may be value in comparing this header to traceparent, so that readers aren't left confused about why both exist